.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,99 @@
+# Adapted from https://github.com/metagraph-dev/metagraph/blob/main/.github/workflows/test_and_deploy.yml
+name: Test and Deploy
+
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  MAIN_PYVER: 3.8
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup conda env
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda info -a
+          conda install -q conda-build conda-verify
+          conda env create -n ci -f ./buildscripts/env-py3.8.yml
+          conda activate ci
+          pip install -e .
+      - name: Lint with Black
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate ci
+          black . --check --diff
+      - name: Pytest
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate ci
+          pytest numba_extras
+      - name: Conda Build
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda build --python ${{ env.MAIN_PYVER }} -c defaults -c numba -c conda-forge buildscripts/conda
+          # This doesn't rebuild, but simply computes the name of the file that was previously built
+          OUTPUT=$(conda build --output --python ${{ env.MAIN_PYVER }} -c defaults -c numba -c conda-forge buildscripts/conda)
```

Comment:
Is conda-forge needed in here? I guess yes for `black` versions?

Reply:
yes, for `black`